app/components/Button/Button.jsx

```diff
@@ -145,7 +146,7 @@ export default class Button extends React.Component {
       }
 
       return (
-        <Link className={buttonStyle.getClasses()} to={href}>
+        <Link className={buttonStyle.getClasses()} to={href} onClick={onClick}>
```

Comment:
`Button` has been using `href` and `onClick` separately (i.e. never together), to decide whether to render an anchor or a button. I guess your change shouldn't have any side-effects because we check for `href` first, but I just wanted to double check with you. :)

Reply:
Explained in a comment in `ErrorMessage.jsx`